core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/RulesDownloader.java

```diff
@@ -143,7 +130,13 @@ private static void readPage(Rules.Builder rulesBuilder, Map<String, ActiveRules
         }
         activeRulesBuildersByQProfile.get(qProfileKey).getMutableActiveRulesByKey().put(entry.getKey(), arBuilder.build());
       }
+    }
 
+    for (Entry<String, org.sonarqube.ws.Rules.QProfile> entry : response.getQProfiles().getQProfiles().entrySet()) {
+      System.out.println("ENTRY: " + entry.getValue().getName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS106)